python/cudf/cudf/utils/utils.py

```diff
@@ -363,14 +364,7 @@ def time_col_replace_nulls(input_col):
     null = column.column_empty_like(input_col, masked=True, newsize=1)
     out_col = cudf._lib.replace.replace(
         input_col,
-        column.as_column(
-            Buffer(
-                np.array(
-                    [input_col.default_na_value()], dtype=input_col.dtype
-                ).view("|u1")
-            ),
-            dtype=input_col.dtype,
-        ),
+        make_column_from_scalar(input_col.default_na_value(), 1),
```

Comment:
The canonical way to create a column from arbitrary objects has been `as_column`. Can we call `make_column_from_scalar` from inside `as_scalar` and just use that here?

Reply:
Done 👍